gallery/workbook/microsoft.network-applicationgatewaywebapplicationfirewallpolicies.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
+  "version": "TemplateGallery/1.0",
+  "categories": [
+    {
+      "id": "WAFPolicyWorkbooks",
+      "name": "Network Insights WAF Policy",
+      "templates": [
+        {
+          "id": "Workbooks/Network Insights/ApplicationGatewayWebApplicationFirewallPolicies/AppGwWafDefault",
+          "author": "Microsoft",
+          "name": "Network Insights WAF Policy"

```

Comment:
best practice: there's an optional `description` field here to explain what it is / what it does